tests/test_modal.py

```diff
@@ -92,7 +92,7 @@ async def test_modal_pop_screen():
         await pilot.pause(0.4)
         await app.wait_for_refresh()
         # Check clicking the footer brings up the quit screen
-        await pilot.click(Footer)
+        await pilot.mouse_down(Footer)
```

Comment:
Doesn't this change the nature of the test significantly? IIRC a click occurs on mouse up.

Reply:
Scratch that. I see that the Footer reacts to MouseDown, not Click.